More null pointer dereferences when plugins are off

I run cppcheck against sources and found two more null pointer dereferences identical to issue #32338


src/server/services/wcs/qgswcsdescribecoverage.cpp:62:21: error:inconclusive: Null pointer dereference: describeDocument [nullPointer]
    response.write( describeDocument->toByteArray() );

src/server/services/wms/qgswmsgetcontext.cpp:84:21: error:inconclusive: Null pointer dereference: contextDocument [nullPointer]
    response.write( contextDocument->toByteArray() );



```
--- src/server/services/wcs/qgswcsdescribecoverage.cpp     2019-09-13 15:11:09.000000000 +0300
+++ src/server/services/wcs/qgswcsdescribecoverage.cpp  2019-10-22 19:38:58.515127044 +0300
@@ -57,6 +57,7 @@
     }
 #else
     doc = createDescribeCoverageDocument( serverIface, project, version, request );
+    describeDocument = &doc;
 #endif
     response.setHeader( "Content-Type", "text/xml; charset=utf-8" );
     response.write( describeDocument->toByteArray() );
--- src/server/services/wms/qgswmsgetcontext.cpp   2019-09-13 15:11:09.000000000 +0300
+++ src/server/services/wms/qgswmsgetcontext.cpp        2019-10-22 19:39:59.220199558 +0300
@@ -79,6 +79,7 @@
     }
 #else
     doc = getContext( serverIface, project, version, request );
+    contextDocument = &doc;
 #endif
     response.setHeader( QStringLiteral( "Content-Type" ), QStringLiteral( "text/xml; charset=utf-8" ) );
     response.write( contextDocument->toByteArray() );

```

--- a/src/server/services/wcs/qgswcsdescribecoverage.cpp
+++ b/src/server/services/wcs/qgswcsdescribecoverage.cpp
@@ -57,6 +57,7 @@ namespace QgsWcs
     }
 #else
     doc = createDescribeCoverageDocument( serverIface, project, version, request );
+    describeDocument = &doc;
 #endif
     response.setHeader( "Content-Type", "text/xml; charset=utf-8" );
     response.write( describeDocument->toByteArray() );

--- a/src/server/services/wcs/qgswcsgetcapabilities.cpp
+++ b/src/server/services/wcs/qgswcsgetcapabilities.cpp
@@ -56,6 +56,7 @@ namespace QgsWcs
     }
 #else
     doc = createGetCapabilitiesDocument( serverIface, project, version, request );
+    capabilitiesDocument = &doc;
 #endif
     response.setHeader( QStringLiteral( "Content-Type" ), QStringLiteral( "text/xml; charset=utf-8" ) );
     response.write( capabilitiesDocument->toByteArray() );

--- a/src/server/services/wfs/qgswfsdescribefeaturetype.cpp
+++ b/src/server/services/wfs/qgswfsdescribefeaturetype.cpp
@@ -58,6 +58,7 @@ namespace QgsWfs
     }
 #else
     doc = createDescribeFeatureTypeDocument( serverIface, project, version, request );
+    describeDocument = &doc;
 #endif
     response.setHeader( "Content-Type", "text/xml; charset=utf-8" );
     response.write( describeDocument->toByteArray() );

--- a/src/server/services/wms/qgswmsgetcontext.cpp
+++ b/src/server/services/wms/qgswmsgetcontext.cpp
@@ -79,6 +79,7 @@ namespace QgsWms
     }
 #else
     doc = getContext( serverIface, project, version, request );
+    contextDocument = &doc;
 #endif
     response.setHeader( QStringLiteral( "Content-Type" ), QStringLiteral( "text/xml; charset=utf-8" ) );
     response.write( contextDocument->toByteArray() );

--- a/src/server/services/wmts/qgswmtsgetcapabilities.cpp
+++ b/src/server/services/wmts/qgswmtsgetcapabilities.cpp
@@ -64,6 +64,7 @@ namespace QgsWmts
     }
 #else
     doc = createGetCapabilitiesDocument( serverIface, project, version, request );
+    capabilitiesDocument = &doc;
 #endif
     response.setHeader( QStringLiteral( "Content-Type" ), QStringLiteral( "text/xml; charset=utf-8" ) );
     response.write( capabilitiesDocument->toByteArray() );

